docs/integrations/custom-media-app.md

```diff
@@ -0,0 +1,201 @@
+# Custom Integration: Media App
+
+Want to use Metisa's recommendation engine for your app that isn't quite an e-commerce store? Use this article to integrate your web app to take full advantage of Metisa, and gain access to personalized item Recommendations for all your users, predictive analytics Insight Reports, emails and more.
+
+A media app is any web-based application that serves content, be it soundtracks, articles, visual assets, or anything in between. You can integrate news sites (eg. The New York Times), music streaming sites (eg. Spotify), stock photography sites (eg. Shutterstock) with Metisa, as a few examples.
+
+This article is intended for an audience with some basic technical knowledge of HTML and JavaScript.
+
+## Contents
+
+* [Benefits of using Metisa for your app](#benefits)
+* [Prerequisites](#prerequisites)
+* [Key concepts](#concepts)
+* [Install](#install)
+* [Embed recommendations](#recommendations)
+
+## <a id="benefits"></a> Benefits of using Metisa for your app
+
+Metisa started out with e-commerce. However, we've received numerous enquiries about using Metisa's recommendation engine for apps that are not quite online shops. This article addresses that need.
+
+The benefits of integrating your media app with Metisa are:
+* Ability to automatically recommend items to your users, personalized for every individual
+* Recommend items on your website as well as embedded in emails
+* No need to export and dump data into Metisa; it starts working immediately after integration setup
+* Recommendations get better with time, as more data means better machine learning
+
+Can't wait already? Read on to setup your integration in the following sections!
+
+
+## <a id="prerequisites"></a> Prerequisites
+
+To integrate your media app with Metisa:
+
+1. **You must have a Metisa account.** Custom integrations are available to all Metisa users, whether you're on a Free or Paid tier. Sign up [here](https://askmetisa.com/create).
+2. **You must have basic technical know-how.** You will be fine if you're comfortable navigating files and inserting code blocks, and some light coding with guidance.
+3. **Your app must use a template rendering engine.** EJS, Liquid, Django are some examples. This installation rides on your app's template tags to obtain data to send to Metisa servers.
+
+On the last point, if you are a developer, you may be able to customize the way data is sent to Metisa  servers by fiddling with the [Metisa JavaScript SDK](https://github.com/altitudelabs/metisa-js).
```

Comment:
1. extra space --> Metisa  servers
2. metisa-js docs also need a bit of cleanup
"Metisa empowers businesses ..." is incomplete on `https://github.com/altitudelabs/metisa-js/blob/master/doc/README.md`